frontend/src/components/static_pages/about.css

```diff
@@ -12,13 +12,21 @@
   margin-bottom: 20px;
 }
 .images {
+ margin: 0;
+ margin-bottom: 40px;
+ margin-left: 120px;
```

Comment:
these styles are going to be overwritten by the styles below this.

in CSS the last styles always win